### DISCUSSION TITLE
request http://xxx:12800/v3/segment errorcode:405

### DISCUSSION THREAD

**Discussion by huangyedie** [SkyAPM/SkyAPM-dotnet#561] [Q&A]
Requesting the OAP interface to write data on its own and reporting an error

requestBody:{"TraceId":"a12ff60b-5807-463b-a1f8-fb1c8608219e","TraceSegmentId":"a12ff60b-5807-463b-a1f8-fb1c8608219e","Spans":[{"SpanId":1,"ParentSpanId":0,"StartTime":638223951265028760,"EndTime":638223951265029357,"Refs":[],"OperationName":"OperationName2","Peer":"","SpanType":2,"SpanLayer":1,"ComponentId":0,"IsError":false,"Tags":[],"Logs":[],"SkipAnalysis":false},{"SpanId":0,"ParentSpanId":-1,"StartTime":638223951265030800,"EndTime":638223951265030803,"Refs":[],"OperationName":"OperationName1","Peer":"","SpanType":2,"SpanLayer":1,"ComponentId":0,"IsError":false,"Tags":[],"Logs":[],"SkipAnalysis":false}],"Service":"test","ServiceInstance":"ServiceInstance","IsSizeLimited":false}

Can't I request it directly?

reference：
https://skywalking.apache.org/docs/main/next/en/api/trace-data-protocol-v3/#post-httplocalhost12800v3segment

**Comment by huangyedie** [comment]
response：
{Method: POST, RequestUri: 'http://xxxx:12800/v3/segment', Version: 1.1, Content: System.Net.Http.ByteArrayContent, Headers:
{
Content-Type: application/json
Content-Length: 678
}}

"HTTP method POST is not supported by this URL"
http://xxx:12800/v3/segment
Is there a problem with this?

**Comment by liuhaoyang** [comment]
Please ask this question to [apache/skyWalking repo](https://github.com/apache/skywalking/issues/new/choose)
